src/main/python/resources/data_files/json_data/VEP104.json

```diff
@@ -5,25 +5,25 @@
     "GC": 0.42,
```

Comment:
Improve structure of python-code/resources?
- Python code in `/src/main/python/[etc...]`
- Resources in `/src/main/resources/[etc...]`